python/lsst/jointcal/jointcal.py

```diff
@@ -77,6 +90,7 @@ class JointcalConfig(pexConfig.Config):
         doc = "How to select sources for cross-matching",
         default = "astrometry"
     )
+    # TODO: CmdLineTask has a profile thing built-in: can we tie in to that?
```

Comment:
You should absolutely use the `lsst.pipe.base.cmdLineTask.profile` context manager.
You should not be setting operational parameters like profiling through `Config`.